colin/cli/colin.py

```diff
@@ -48,6 +48,7 @@ def cli():
 @click.command(name="check",
                context_settings=CONTEXT_SETTINGS)
 @click.argument('TARGET', type=click.STRING)
+@click.argument('PARENT_TARGET', type=click.STRING, required=False)
```

Comment:
It would be better if we change CLI interface a little bit.

This version relies on optional positional parameter
```
sh# colin checks -h
Usage: colin checks [OPTIONS] TARGET [PARENT_TARGET]

  Check the image/dockerfile (default).
```

But optional parameters should not be positional. Named parameter fits such use-case better IMHO
e.g. --parent-target

@TomasTomecek @lachmanfrantisek
What do you think ?

Reply:
Yup, sounds good.

Reply:
changed